doc/src/fix_propel_self.rst

```diff
@@ -8,94 +8,165 @@ Syntax
 
 .. parsed-literal::
 
-   fix ID group-ID propel/self mode magnitude keyword values ...
+   fix ID group-ID propel/self magnitude keyword values
 
 * ID, group-ID are documented in :doc:`fix <fix>` command
 * propel/self = style name of this fix command
-* mode = velocity or quat
-* magnitude = magnitude of the active force
-* one or more keyword/value pairs may be appended to args
-* keyword = *types*
+* magnitude = magnitude of self-propulsion force
+* one (and only one) keyword/value pair must be appended to args
+* keyword = *dipole* or *velocity* or *quat*
+
+  .. parsed-literal::
+
+       *dipole* value = none = apply force along dipole direction
+       *velocity* value = none = apply force along velocity direction
+       *quat* values = direction vector *sx* and *sy* and *sz*
+        *sx* = x component of force direction in ellipsoid frame
```

Comment:
the direction in the ellipsoid frame should be added as an optional keyword, say, "qvector \<sx\> \<sy\> \<sz\>" or similar.

Reply:
Okay, I have updated the doc accordingly. I wasn't 100% on syntax so let me know if I need to make further changes.